mm2src/adex_cli/src/scenarios/init_mm2_cfg.rs

```diff
@@ -226,9 +229,12 @@ impl Mm2Cfg {
 
     #[inline]
     fn inquire_rpcip(&mut self) -> Result<()> {
+        let default_ip: Ipv4Addr = DEFAULT_RPC_IP.parse().map_err(|error| error_anyhow!("Failed pass default rpcip: {error}"))?;
```

Comment:
Why not making `DEFAULT_RPC_IP` type `Ipv4Addr` instead of `&str`?

Reply:
done thanks.